ethcontract/src/lib.rs

```diff
@@ -165,6 +165,8 @@ pub mod dyns {
     pub type DynAllEventsBuilder<E> = AllEventsBuilder<DynTransport, E>;
 }
 
+#[allow(unknown_lints)]
+#[allow(clippy::mixed_attributes_style)]
```

Comment:
Can be removed since I turned the hidden doc comments into regular comments since they are hidden anyway.